draft-ietf-masque-h3-datagram.md

```diff
@@ -334,8 +334,14 @@ forward Capsules with an unknown Capsule Type without modification.
 Endpoints which receive a Capsule with an unknown Capsule Type MUST silently
 drop that Capsule and skip over it to parse the next Capsule.
 
-By virtue of the definition of the data stream, the Capsule Protocol is not in
-use on responses unless the response includes a 2xx (Successful) status code.
+By virtue of the definition of the data stream:
+
+* The Capsule Protocol is not in use on responses unless the response includes a
+  2xx (Successful) status code.
```

Comment:
Strictly, the capsule protocol isn't in use at all unless the response is 2xx right?

Maybe just simplify this statement further.
```suggestion
* The Capsule Protocol is not in use unless the response includes a
  2xx (Successful) status code.
```